include/benchmark/benchmark.h

```diff
@@ -375,9 +375,12 @@ class Counter {
 
   double value;
   Flags flags;
+  std::pair<int /*Num*/, int /*Denom*/> thousand;
```

Comment:
Hmm, if it to be an enum, should i just add value `k1024` to the `Flags` enum?

Reply:
I think it's an orthogonal feature so i'd add a OneK enum with two values.

Reply:
Ok, sounds good.